.gitignore

```diff
@@ -7,3 +7,5 @@ tmp/
 out/
 *.pyc
 __pycache__
+utils/oppo_*
+utils/vmlinux*
```

Comment:
Thanks for this fix, totally forgot to add these directories into .gitignore :thinking: